### PR TITLE
27112: Updated NEWS.rst

### PR DESCRIPTION
**Pull request by cmoesgaard**
- [ ] Have you updated the documentation?
- [ ] Have you performed a smoke test of the application?
- [ ] Have you written tests for your changes?
- [ ] Have you updated the release notes?

### CODE REVIEW

File: NEWS.rst [hunk 1]
```diff
@@ -36,14 +39,14 @@ Bug fixes
 ---------
 * #25405: Submit button for create new and edit modals for organisation 
   units and employees is no longer disabled if the form is invalid
-* #25028: Timeachine is working again.
+* #25028: Timemachine is working again.
```
Comment (danchr) [inline NEWS.rst]:
```suggestion
* #25028: Time machine is working again.
```

File: NEWS.rst [hunk 2]
```diff
@@ -36,14 +39,14 @@ Bug fixes
...
 * #25579: Address race condition when quickly switching between units
   in the tree view at the left.
 * #25186: Hidden person input for create employee manager.
 * #25690: Ignore spacing in address type input field.
 * #26368: Validation no longer prevents adding an association if it
   duplicates another *inactive* association.
-* #25704 Added max-width to table columns.
+* #25704: Added max-width to table columns.
```
Comment (danchr) [inline NEWS.rst]:
```suggestion
* #25704: Set ``max-width`` toon detail view table columns to ensure consistent alignment.
```

File: NEWS.rst [hunk 3]
```diff
@@ -15,6 +15,9 @@ New features
   for disabling certain validations.
 * #25409: Added backend support for specifying visibility for phone number
   address objects.
+* #25706: Added more meaningful error message when editing Address
```
Comment (danchr) [inline NEWS.rst]:
```suggestion
* #25706: Added more meaningful error message when editing addresses.
```

File: NEWS.rst (hunk 2 above)
Comment (cmoesgaard) [reply NEWS.rst]: toon?

Comment (danchr) [reply NEWS.rst]: Fat fingers

**Pull request merged**